R/ducktbl.R

```diff
@@ -76,8 +76,22 @@
 #'
 #' @param ... For `duckdb_tibble()`, passed on to [tibble()].
 #'   For `as_duckdb_tibble()`, passed on to methods.
-#' @param .funnel,funnel Logical, whether to create a funneled duckplyr frame.
-#'   See the section "Funneling" for details.
+#' @param .funnel,funnel Either a logical:
+#'   - Set to `TRUE` to return a funneled data frame.
+#'   - Set to `FALSE` to return an unfunneled data frame.
+#'
+#' Or a named vector with at least one of
+#'   - `cells` (numeric)
+#'   - `rows` (numeric)
+#'
+#' to allow materialization for data up to a certain size,
+#'   measured in cells (values) and rows in the resulting data frame.
+#'
+#' If `cells` is specified but not `rows`, `rows` is `Inf`.
+#' If `rows` is specified but not `cells`, `cells` is `Inf`.
+#'
+#' The default is to inherit the funneling of the input.
+#'   see the "Funneling" section.
```

Comment:
I forgot that this section needs to go or be shortened drastically, pointing to the vignette.

Reply:
You made a comment about this but I'll track it in an issue.